app/models/site_config.rb

```diff
@@ -41,42 +39,53 @@ class SiteConfig < RailsSettings::Base
     members: "members@dev.to"
   }
 
+  # Email digest frequency
+  field :periodic_email_digest_max, type: :integer, default: 0
+  field :periodic_email_digest_min, type: :integer, default: 2
+
+  # Google Analytics Reporting API v4
+  # <https://developers.google.com/analytics/devguides/reporting/core/v4>
+  field :ga_view_id, type: :string, default: ""
+  field :ga_fetch_rate, type: :integer, default: 25
+
+  # Images
+  field :main_social_image, type: :string, default: "https://thepracticaldev.s3.amazonaws.com/i/6hqmcjaxbgbon8ydw93z.png"
+  field :favicon_url, type: :string, default: "favicon.ico"
+  field :logo_png, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/devlogo-pwa-512.png"
+  field :logo_svg, type: :string, default: ""
+  field :primary_sticker_image_url, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/rainbowdev.svg"
+
+  # Mascot
+  field :mascot_user_id, type: :integer, default: 1
+  field :mascot_image_url, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/sloan.png"
+  field :mascot_image_description, type: :string, default: "Sloan, the sloth mascot"
+
   # Meta keywords
   field :meta_keywords, type: :hash, default: {
     default: nil,
     article: nil,
     tag: nil
   }
 
-  # Authentication
-  field :authentication_providers, type: :array, default: Authentication::Providers.available
-
-  # Broadcast
-  field :welcome_notifications_live_at, type: :date
```

Comment:
Looks like this is why those specs are breaking. Should the specs be removed or should this line be put back?

Reply:
Oh gosh thank goodness the specs picked up that up. Thanks @jacobherrington! I'm adding it back.